### PR TITLE
Dynamic sync for contributors page

### PR DESCRIPTION
**Pull request by harry-hov**
# What i did ?
### 1. Contributors page is now in sync with Github and show realtime data(Followers, Following, etc).
screenshot 55

### 4. Now its simple to add Contributors in "Contributors page". Type one line and you are done.
`<div class="github-card" data-github="git_username" data-theme="medium"></div>`

### CODE REVIEW

**Comment by monsij** [comment]
Hope that you have followed the contribution guidelines :-)

(Powered by [GitMate.io](https://gitmate.io))

**Comment by monsij** [comment]
@harry-hov Can you add the info about the number of commits the user has made .. And the no.of reviews made.Remove the number of repos and following and replace them with the above.

**Comment by monsij** [comment]
@harry-hov Can you add the info about the number of commits the user has made .. And the no.of reviews made.Remove the number of repos and following and replace them with the above.

**Comment by harry-hov** [comment]
> @harry-hov Can you add the info about the number of commits the user has made .. And the no.of reviews made.Remove the number of repos and following and replace them with the above.

It cant do this without Github API keys.

**Comment by monsij** [comment]
Ok so should we go for it ?

**Comment by harry-hov** [comment]
Yeah. so I can start working on another UI enhancements.

**Comment by monsij** [comment]
You can leave out the api fields blank

**Comment by monsij** [comment]
Once merged..ill get them ready..do add a dummy text where these keys will go into

**Comment by harry-hov** [comment]
you mean sample for using api tokens ?

> Once merged..ill get them ready..do add a dummy text where these keys will go into

**Comment by monsij** [comment]
Will merge this for now

**Comment by monsij** [comment]
Open a separate issue for the remaining....squash your commits to get things ready

**Comment by harry-hov** [comment]
done, you can merge changes now

**Comment by monsij** [comment]
What about these? Are these required to be committed?

> Also let us know if the files under `__pycache__` is required to be commited or they are just cache files

**Comment by harry-hov** [comment]
> What about these? Are these required to be committed?
> 
> > Also let us know if the files under `__pycache__` is required to be commited or they are just cache files

nope. they are just cache files

**Comment by harry-hov** [comment]
let me remove those cache files and resolve conflict.

File: pro1/blog/templates/blog/gitCard/medium.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+            <div class="header {type}"></div>
+            <a class="avatar" href="https://github.com/{login}">
+                <img src="{avatar_url}&s=80" alt="{name}">
+            </a>
+            <div class="content">
+                <h1>{name}</h1>
+                <p>@{login}</p>
+                <ul class="status">
+                    <li><a href="https://github.com/{login}?tab=repositories"><strong>{public_repos}</strong>Repos</a></li>
+                    <li><a href="https://gist.github.com/{login}/following"><strong>{following}</strong>Following</a></li>
```
Comment (monsij) [inline pro1/blog/templates/blog/gitCard/medium.html]: Correct the link for the `following`

**Comment by harry-hov** [comment]
please take a look. @monsij

**Comment by monsij** [comment]
"gitmate-bot rebase"

**Comment by monsij** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by monsij** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by monsij** [comment]
ack `6c69a11`

**Comment by monsij** [comment]
This is the last time @harry-hov that i'm merging from your master branch of fork. Please take a note of it from next time. 😅

**Pull request merged**